tests/suites/test_suite_mpi.function

```diff
@@ -50,6 +50,181 @@ int mbedtls_test_mpi_miller_rabin_determinizer( void* state,
  * END_DEPENDENCIES
  */
 
+/* BEGIN_CASE depends_on:MBEDTLS_CHECK_PARAMS:!MBEDTLS_PARAM_FAILED_ALT */
+void mpi_invalid_param( )
+{
+    mbedtls_mpi X;
+    const char *s_in = "00101000101010";
+    char s_out[16] = { 0 };
+    unsigned char u_out[16] = { 0 };
+    unsigned char u_in[16] = { 0 };
+    size_t olen;
+    mbedtls_mpi_uint mpi_uint;
+
+    TEST_INVALID_PARAM( mbedtls_mpi_init( NULL ) );
+    TEST_INVALID_PARAM_RET( MBEDTLS_ERR_MPI_BAD_INPUT_DATA,
+                            mbedtls_mpi_grow( NULL, 42 ) );
```

Comment:
Shouldn't we also have a test for nblimbs = `MBEDTLS_MPI_MAX_LIMBS+1`?

Reply:
I deliberately omitted it because it's checked unconditionally, and not by parameter validation.

Reply:
Then  that would be in the second group of 'always valid' tests.

Reply:
We should add this test, but at this point I can accept the omission.